docs/resources/integration_gcp_sts_account.md

```diff
@@ -0,0 +1,49 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "datadog_integration_gcp_sts_account Resource - terraform-provider-datadog"
+subcategory: ""
+description: |-
+  Provides a Datadog IntegrationGcpStsAccount resource. This can be used to create and manage Datadog integrationgcpsts_account.
+---
+
+# datadog_integration_gcp_sts_account (Resource)
+
+Provides a Datadog IntegrationGcpStsAccount resource. This can be used to create and manage Datadog integration_gcp_sts_account.
+
+## Example Usage
+
+```terraform
+# Create new integration_gcp_sts_account resource
+
+resource "datadog_integration_gcp_sts_account" "foo" {
+  client_email    = "service-account@example.com"
+  host_filters    = ["filter_one", "filter_two"]
+  automute        = true
+  is_cspm_enabled = true
+}
+```
+
+<!-- schema generated by tfplugindocs -->
+## Schema
+
+### Required
+
+- `client_email` (String) Your service account email address.
+
+### Optional
+
+- `automute` (Boolean) Silence monitors for expected GCE instance shutdowns.
+- `host_filters` (Set of String) Your Host Filters.
```

Comment:
```suggestion
- `host_filters` (Set of Strings) Your Host Filters.
```

Reply:
Thanks for the feedback. This part of the documentation is auto generated and we cannot change it without changing the underlying package.